docs/languages/powershell.md

```diff
@@ -1,31 +1,39 @@
 # Editing PowerShell in Visual Studio Code
 
-PowerShell is a task-based command line shell and scripting language built on the .NET framework.
-Powershell provides a powerfull toolset for administrators on any platform.
+PowerShell is a task-based command line shell and scripting language built on .NET .
 
-The PowerShell extension provides rich [PowerShell Language Support](https://github.com/PowerShell/vscode-powershell)
+PowerShell provides a powerful toolset for administrators on any platform.
 
-## Install the PowerShell extension on Visual Studio Code
+[The PowerShell Extension](https://github.com/PowerShell/vscode-powershell) provides rich language Support and IDE capabilities such as completions, definition tracking and linting analysis for Windows PowerShell versions 3, 4, 5 and 5.1 as well as all versions of PowerShell Core.
 
-### Installing the Extension
+## Install the PowerShell Extension on Visual Studio Code
 
-The official extension can be installed by following the steps in the
+The official PowerShell Extension can be installed by following the steps in the
 [Visual Studio Code Docs](https://code.visualstudio.com/docs/editor/extension-gallery)
 or by going directly to the Marketplace and clicking the [Install Button](vscode:extension/ms-vscode.PowerShell)
 
-You can also install the Extension from within Visual Studio code by
-browsing
+You can also install the PowerShell Extension from within Visual Studio Code by
+browsing the Extension Marketplace under the Marketplace view in the Activity Bar for extensions.
+You can access this with the shortcut <kbd>ctrl</kbd>-<kbd>Shift</kbd>-<kbd>x</kbd> or by clicking on the
 
-![Extension Browser](images/powershell/extensionBrowser.png)
+ Marketplace Icon:
 
- for extensions `(ctl+Shift+x)`  and searching for PowerShell. Select the PowerShell extension for details
+![Marketplace Activity Icon](./images/powershell/extensionBrowser.png)
 
- ![PowerShell Extension Details](images/powershell/PowerShellExtension.png)
+Search the Marketplace View for "PowerShell" and select the PowerShell Extension,
+Which is denoted by this Icon:
 
- Alternatively you can install from PowerShell command line
- this works on all platforms
+![PowerShell Extension](./images/powershell/pwshVsixIcon.png)
 
-    code --install-extension ms-vscode.powershell
+ ## Search from the Command line
+
+ Alternatively, the PowerShell Extension can be installed from any command line (including PowerShell, CMD, bash) on all platforms with the following command
+ 
+ ```code --install-extension ms-vscode.powershell```
```

Comment:
Ah, I think @SteveL-MSFT meant a code block like this:
```
code --install-extension ms-vscode.powershell
```
Done like this:
````
```
code --install-extension ms-vscode.powershell
```
````

Reply:
Ok I'll get this fixed for review this weekend.